cpu/xmc1000/Makefile.include

```diff
@@ -0,0 +1,7 @@
+export CPU_ARCH = cortex-m0
+
+# only xmc1100 supported for now
+MODEL = $(shell echo $(CPU_MODEL)|tr 'a-z' 'A-Z')
+export CFLAGS += -DCPU_$(MODEL)
```

Comment:
`-DCPU_MODEL_$(MODEL)` is already defined in `Makefile.include.cortexm_common` and this one seems not to be used.